app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,11 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body style="height: 100%!important;">
```

Comment:
Avoid inline-style (I did it due to SweetAlert2 classes overrode mines, but there are better ways)

Reply:
Yeah, actually, i forgot to erase it hehe I was just trying to figure out what you did (with our talked i understood)